src/index.html

```diff
@@ -19,18 +19,21 @@ <h1><a class="identity-logo" href="#">Creative Commons</a></h1>
         <button class="expand-menu">Menu</button>
         <nav class="primary-menu">
             <ul>
-                <li><a href="">Who We Are</a></li>
-                <li><a href="">What We Do</a></li>
-                <li><a href="">Licenses and Tools</a></li>
-                <li><a href="">Blog</a></li>
-                <li><a href="">Support Us</a></li>
+                <li><a href="#">Who We Are</a></li>
+                <li><a href="#">What We Do</a></li>
+                <li><a href="#">Licenses and Tools</a></li>
+                <li><a href="#">Blog</a></li>
+                <li><a href="#">Support Us</a></li>
```

Comment:
I don't hate this idea, but since the standard way to denote an empty anchor is the single hash, rather than an anchor, which attempts to jump to that ID on the page. I'm concerned about taking this approach. I'm gonna make a note of it to consider it further, but it feels a little like optimizing a process that doesn't exist yet with something slightly off standard. But I really do appreciate the idea.